presto-main/src/main/java/io/prestosql/operator/scalar/MathFunctions.java

```diff
@@ -607,11 +607,47 @@ public static long random(@SqlType(StandardTypes.BIGINT) long value)
         return ThreadLocalRandom.current().nextLong(value);
     }
 
+    @Description("a pseudo-random number between m and n (exclusive")
+    @ScalarFunction(value = "random", alias = "rand", deterministic = false)
+    @SqlType(StandardTypes.TINYINT)
+    public static long rangeRandomTinyInt(@SqlType(StandardTypes.TINYINT) long initialValue, @SqlType(StandardTypes.TINYINT) long finalValue) {
```

Comment:
In Presto we use Airlift codestyle -- please follow https://github.com/airlift/codestyle for instructions

In particular this means:
```suggestion
    public static long rangeRandomTinyInt(@SqlType(StandardTypes.TINYINT) long initialValue, @SqlType(StandardTypes.TINYINT) long finalValue)
    {
```